publishing/CSS.html

```diff
@@ -48,6 +48,18 @@
                 <li>
                     The <span data-feature="CSS Grid 1"><a data-featureid="css-grid-1">CSS Grid 1</a></span> module is not specific to paginated media, but is nonetheless noteworthy as it enables advanced layouts often found in print publications. 
                 </li>
+
+                <li data-feature="CSS Rhythms">
+                    The <a data-featureid="css-rhythm-1">CSS Rhythmic Sizing</a> module contains CSS features for aligning content size to multiple of unit size.
+                </li>
+
+                <li data-feature="CSS Line Grid">
+                        The <a data-featureid="css-line-grid-1">CSS Line Grid</a> module contains CSS features for aligning content to a baseline grid.
+                </li>
+    
+                <li data-feature="CSS Fonts">
+                    The <a data-featureid="css-font-4">CSS Font 4</a> module introduces a number of features like <code>font-stretch</code>, <code>font-synthesis</code> (oblique, which a lot of publishers do in their DTP tools when an italic is not available), OTF, color fonts, variable fonts.
```

Comment:
You'll need to update the reference to "css-fonts-4" here once you've changed the name of the file.